MeetU/MeetU/API/FollowsController.cs

```diff
@@ -24,6 +31,42 @@ public IQueryable<Follow> GetFollows()
             return db.Follows;
         }
 
+        // GET: api/Following/5
+        [HttpGet]
+        [Route("api/Following/{userId}")]
+        public IQueryable<FollowingOrFollowedView> GetProfileByFollowingUserId(string userId)
+        {
+            return
+                from p in db.Profiles
```

Comment:
just recommend you guys use method syntax rather than query syntax. Method syntax is way more simpler and easier to read, write and debug.
https://msdn.microsoft.com/en-us/library/bb397947.aspx
http://stackoverflow.com/questions/2767709/c-sharp-joins-where-with-linq-and-lambda